man/systemd.system-credentials.xml

```diff
@@ -247,6 +247,21 @@
         <xi:include href="version-info.xml" xpointer="v254"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>journald.forward_to_syslog</varname></term>
+        <term><varname>journald.forward_to_kmsg</varname></term>
+        <term><varname>journald.forward_to_console</varname></term>
+        <term><varname>journald.forward_to_wall</varname></term>
```

Comment:
hmm, what's the usecase for these? Do we actually need them? 

I mean, there's a discussion to be had what credentials shall be used for. To me it's not so much as a way to carry individual config options but to pass along more complex payload. Hence I am not sure we should have fine-grained options like this. What's the precise usecase for this?

Reply:
So I was originally going to add forward_to_socket as an option.
But I think it is cleaner to simply forward if the socket connection exists.

There was some discussion that credentials would be preferred so I added code to make the existing forwarding options available via credentials but they are now vestigial as forward_to_socket isn't being added.

I will rework it to just add the forward_address credential instead when it is out of draft.